lib/chef/resource/openssl_dhparam.rb

```diff
@@ -72,6 +71,15 @@ class OpensslDhparam < Chef::Resource
             end
           end
         end
+        if ::File.exist?(new_resource.path)
+          file new_resource.path do
+            action :create
+            owner new_resource.owner unless new_resource.owner.nil?
+            group new_resource.group unless new_resource.group.nil?
```

Comment:
@lamont-granquist says in both of these resources we can simply do:

```
owner new_resource.owner
group new_resource.group
```

(and drop the conditional)